### DISCUSSION TITLE
USWNT

### DISCUSSION THREAD

**Discussion by bsantero** [vasqued2/ha-teamtracker#136] [Q&A]
I'm having some trouble configuring my (not-so-fringe but still niche) teams.

Does anyone have any idea about conguring the USWNT? The docs are not particularly clear with this and others say AFLW.

**Comment by vasqued2** [comment]
There is no single API for USWNT.  There are different ones for different events.

yYou will have to set up a custom API configuration.  Easiest way is to view the event on ESPN and look at the URL and get the league_path from there.

https://www.espn.com/soccer/scoreboard/_/league/concacaf.w.gold

so league path for concacaf is concacaf.w.gold

when they play friendlies or World Cup it will be different.
